2019_01_18_ListsDemo/src/MainPersons.java

```diff
@@ -72,7 +72,7 @@ public static void main(String[] arg) {
 		return res;
 	}
 
-public static List<Address> getAddressesByPersonName(List<Person> persons, Person person){
+public static List<Address> getAddressesByOnePersonName(List<Person> persons, Person person){

```

Comment:
this pr is only about renaming the method, which does not seem correct though since you still pass Person as a parameter, not its name.